src/app.py

```diff
@@ -41,41 +41,41 @@ def put_comment(id_comment):
     return comment_schema.jsonify(comment)
 
 # Function API REST DELETE delete comment from database with id parameter
-@app.route('/comment_DEL/<int:id_comment>', methods=['DELETE'])
-def delete_comment(id_comment):
-    comment = Comment.query.filter_by(id_comment = id_comment).first()
+@app.route('/api/v1/comment/<int:id>', methods=['DELETE'])
+def delete_comment(id):
+    comment = Comment.query.filter_by(id_comment = id).first()
     db.session.delete(comment)
     db.session.commit()
     print(comment)
     return comment_schema.jsonify(comment)
 
-# Function API REST POST create new response database
-@app.route('/response_POST/<int:id_comment>', methods=['POST'])
-def create_response(id_comment):
+# Function API REST POST create new response database with id comment parameter
+@app.route('/api/v1/response/<int:id>', methods=['POST'])
```

Comment:
please use plural `/api/v1/responses` , same for the rest of methods

Reply:
in app route with method GET
@app.route('/api/v1/responses/<int:id>', methods=['GET'])
exist diferent parameter reference int:id require, first to references at id_comment and second to references at id_response
how to diferences in the routes because both is same routes.

Reply:
oh! got it, a response belongs to a comment right? if that is the case we need to move the responses under comments like 
`/api/v1/comments/<comment_id>/responses` GET, POST 
`/api/v1/comments/<comment_id>/responses/<id>` GET, PUT, DELETE